packages/ripple-nuxt-tide/modules/data-driven-component/components/TideMapPreventionFamilyViolence/TideMapCard.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="tide-map-card" v-on:click="cardClicked">
+    <div class="tide-map-card-header">
+      <rpl-icon
+        v-if="isArea"
+        class="card-map-marker"
+        symbol="map_marker"
+        color="mid_neutral_1"
+        size="m"
+      />
+      <b>{{item.title}}</b>
+      <div class="card-arrow">
+        <rpl-icon symbol="arrow_right_secondary" color="dark_neutral" size="s"/>
+      </div>
+    </div>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import { RplIcon } from '@dpc-sdp/ripple-icon'
+
+export default {
+  name: 'TideMapCard',
+  props: {
+    item: Object,
+    isArea: Boolean
+  },
+  components: {
+    RplIcon
+  },
+  methods: {
+    cardClicked () {
+      this.$emit('item-clicked', this.item)
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+@import '~@dpc-sdp/ripple-global/scss/settings';
+@import '~@dpc-sdp/ripple-global/scss/tools';
+
+$tide-map-card-background-color: rpl-color('white') !default;
+$tide-map-card-text-color: rpl-color('primary') !default;
+// Found the following on CardContent.vue
+$rpl-card-content-border-color: rpl_color('mid_neutral_1') !default;
+$rpl-card-content-border: 1px solid $rpl-card-content-border-color !default;
+$rpl-card-content-border-radius: rem(4px) !default;
+
+.tide-map-card {
+  cursor: pointer;
+  margin-top: 5px;
```

Comment:
If possible, please use our [spacing variable](https://github.com/dpc-sdp/ripple/blob/93ba6b95dcfd4a3a664e02097772f962b3968f50/packages/ripple-ui-components/components/Atoms/Global/scss/settings/_variables.scss#L19) for spacing. 

Our SDP design system is trying to make the smallest space unit of `4px`, and we are using `rem` instead of `px`. So if not have to be `5px`, this can be `margin-top: $rpl-space;`. For place you need `8px`, can use `$rpl-space-2`. For `20px`, can be `$rpl-space * 5`. For other specific size can't use our variable, use `rem()`. e.g `rem(125px)`.

That will make it easier if you want to change spacing for all components.

Reply:
all padding and margins switched `$rpl-space`